vendor/vendor.json

```diff
@@ -110,7 +110,7 @@
 			"revisionTime": "2017-09-28T14:21:59Z"
 		},
 		{
-			"checksumSHA1": "fqejS2llZXw3gLnOYhg7pcSlY+Q=",
+			"checksumSHA1": "Yfj2ZrgRrklXrshCM1RxH5pvUY8=",
```

Comment:
This change should not be part of the PR.

Reply:
hmmm... not sure how that changed